apps/mobile/__tests__/hooks/use-adoptions/index.test.ts

```diff
@@ -0,0 +1,89 @@
+import { AdoptionType } from "@animavita/models";
```

Comment:
I don't think we need to test this file cause this is just a wrapper around `react-query`. As long as you don't have custom logic in it, no need to test.

Reply:
Just remove it?

Reply:
Yes!